your-first-meeting.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Attending your first PR Review Club Meeting
+---
+
+## Attending your first PR Review Club Meeting
+
+If you’re thinking about attending PR Review Club, welcome!  We love new
+participants. Below is information to help you get started as a first timer.
+
+The club meets on {{ site.meeting_day }}s at {{ site.meeting_time }} on the
+`#bitcoin-core-pr-reviews` IRC channel on
+[freenode](https://webchat.freenode.net/). Every week, a Bitcoin protocol
+developer will host a 60 minute discussion on a Bitcoin Core pull request. [The
+code of conduct](https://bitcoincore.reviews/code-of-conduct) details the
+behavior we expect from all participants.
+
+<strong>If you’re new to IRC (Internet Relay Chat),</strong> it has its
+own little culture and etiquette.
+
+The basics:
+
+- Follow the Golden Rule: treat others as you’d like to be treated.
+- Assume the best in others (tone is hard over text)
+- Talk in full sentences, but don't stress too much about grammar or
+  capitalization!
+- To respond directly to someone, just start your message with their full
+  nick and a colon, like this: <strong>otherperson: thanks for the
+  help!</strong> There's no need to @ them.
+- Just observing (sometimes called lurking) is completely fine.
+
+You’ll get the most out of Review Club if you’ve reviewed the PR going into the
+meeting, but it’s definitely not mandatory in order to attend. You can always
+show up as an observer. If you're worried (don't be), read a log of one of the
+[past PR Review Clubs](https://bitcoincore.reviews/meetings/). You'll see it's
+very casual.
+
+If you’d like to more actively participate, you should clone the [Bitcoin
+repository](https://github.com/bitcoin/bitcoin). You should also check out and
+build the PR branch and run all tests. Take time to review the code changes and
```

Comment:
suggest adding the word "locally" after "branch"

I've heard that checking out the PR branch locally (and reviewing the diff locally rather than on GitHub) is an area of uncertainty for people who are new to git and this process and wrote a little about it in https://jonatack.github.io/articles/how-to-review-pull-requests-in-bitcoin-core#pull-down-the-code-locally